packages/react/src/components/Masthead/Masthead.js

```diff
@@ -97,46 +123,60 @@ const Masthead = ({ navigation, ...mastheadProps }) => {
   return (
     <HeaderContainer
       render={({ isSideNavExpanded, onClickSideNavExpand }) => (
-        <>
-          <Header aria-label="IBM" data-autoid="masthead">
-            <SkipToContent />
-            <HeaderMenuButton
-              aria-label="Open menu"
-              data-autoid="masthead__hamburger"
-              onClick={onClickSideNavExpand}
-              isActive={isSideNavExpanded}
-            />
-            <div className={className}>
-              <a href="https://ibm.com">
-                <IbmLogo data-autoid="masthead__logo" />
-              </a>
-            </div>
-            <div className={`${prefix}--header__search`}>
-              {navigation !== false ? (
-                <MastheadTopNav {...mastheadProps} navigation={mastheadData} />
-              ) : null}
-              <MastheadSearch />
-            </div>
+        <div
+          className={`${prefix}--masthead ${mastheadSticky}`}
```

Comment:
You mean this?
https://github.com/kennylam/ibm-dotcom-library/blob/feat/dotcomshell-options/packages/react/src/components/Masthead/Masthead.js#L88

Reply:
Yeah, what do you think?

```javascript
<div className={cx(`${prefix}--masthead`, mastheadSticky)} />
```

Reply:
Hm, not sure, since I'm already calling `cx` to set the class name vars to begin with. I don't mind if we move to this syntax going forward, and we change it in all packages. But changing this one instance now wouldn't make sense since I'm using the current syntax in other files as well.